scripts/utils/generate_ts.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+. $(dirname $0)/commons.sh
+
+print N "This script generates a 'ts' file for the location of the Mozilla VPN client"
+print N ""
+
+cd $(dirname $0)/../.. || die
+
+printn Y "Branch name: "
+BRANCHNAME="$(git symbolic-ref HEAD 2>/dev/null)"
+BRANCHNAME=${BRANCHNAME##refs/heads/}
+print G "$BRANCHNAME"
+
+printn Y "Caching the dep scripts... "
+mkdir -p cache || die
+cp scripts/utils/generate_strings.py cache || die
+print G "done."
+
+printn Y "Generating strings... "
+python cache/generate_strings.py -o translations/generated -g src/ui/guides -t src/ui/tutorials
+print G "done."
+
+printn Y "Generating a dummy PRO file... "
+cat > translations/generated/dummy.pro << EOF
+HEADERS += l18nstrings.h
+SOURCES += l18nstrings_p.cpp
+SOURCES += ../l18nstrings.cpp
+TRANSLATIONS += translations.ts
+HEADERS += \$\$files(../../src/*.h, true)
+SOURCES += \$\$files(../../src/*.cpp, true)
+RESOURCES += \$\$files(../../src/*.qrc, true)
+HEADERS += \$\$files(../../nebula/*.h, true)
+SOURCES += \$\$files(../../nebula/*.cpp, true)
+RESOURCES += \$\$files(../../nebula/*.qrc, true)
+EOF
+print G "done"
+
+print Y "Generating the main translation file... "
+lupdate translations/generated/dummy.pro -ts translations.ts || die
+
+git branch -r
+for branch in $(git branch -r | grep origin/releases); do
+  printn Y "Importing strings from $branch..."
```

Comment:
Is it just me or I don't see this part of the code running in the action?
https://github.com/mozilla-mobile/mozilla-vpn-client/runs/6738300986?check_suite_focus=true

Also don't see the branch name printed from line 17.

```
022-06-04T12:46:25.9413195Z Branch name: 
2022-06-04T12:46:25.9449174Z Caching the dep scripts... done.
2022-06-04T12:46:26.0604284Z Generating strings... done.
2022-06-04T12:46:26.0616738Z Generating a dummy PRO file... done
2022-06-04T12:46:26.0617327Z Generating the main translation file... 
2022-06-04T12:46:26.1787790Z Info: creating stash file /home/runner/work/mozilla-vpn-client/mozilla-vpn-client/.qmake.stash
2022-06-04T12:46:26.1839241Z lupdate warning: TS files from command line will override TRANSLATIONS in /home/runner/work/mozilla-vpn-client/mozilla-vpn-client/translations/generated/dummy.pro.
2022-06-04T12:46:26.2991234Z Updating 'translations.ts'...
2022-06-04T12:46:26.2997391Z     Found 493 source text(s) (493 new and 0 already existing)
2022-06-04T12:46:26.3022964Z   pull/3664/merge
2022-06-04T12:46:26.3062422Z Remove cache... done.
```